### ISSUE TITLE
MERN_React 6.1 Forms  in React 

### ISSUE THREAD

**Issue by manjeetdhayal** [girlscript/winter-of-contributing#2267]
### Description

- How to make forms in react 
- creating schema for forms 

### Domain

MERN

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

**Comment by Artistic18** [comment]
@ishangupta6412 are you working on this issue?

**Comment by ishangupta6412** [comment]
Yes

**Comment by Artistic18** [comment]
@ishangupta6412 Sure, I was checking on it because there were no updates.

**Comment by AmanBisht01** [comment]
@ishangupta6412 Any update ?

**Comment by ishangupta6412** [comment]
Yes bhaiya I am working on it. My mid sems are going on this week. I will
surely complete it as soon as possible.

**Comment by ishangupta6412** [comment]
https://github.com/girlscript/winter-of-contributing/pulls/ishangupta6412

bhaiya this is the link
